electron/index.cjs

```diff
@@ -52,7 +76,7 @@ const createTray = (window) => {
     {
       label: 'Show',
       click: () => {
-        win.maximize();
+        setWindowBounds(window);
```

Comment:
I'm not sure what Tray() should be in MacOS because I see no menu with "Show" label anywhere. If anyone could test it on Windows and Linux it would be great.